include/labwc.h

```diff
@@ -467,6 +471,8 @@ struct wlr_box output_usable_area_in_layout_coords(struct output *output);
 struct wlr_box output_usable_area_scaled(struct output *output);
 void handle_output_power_manager_set_mode(struct wl_listener *listener,
 	void *data);
+void virtual_output_add(struct server *server, const char *output_name);
```

Comment:
As these functions are in `output.c` their function names ought to begin with `output_`